site/src/components/data/quandl.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: component
+title: Quandl
+component: data/feed/quandl.js
+namespace: data
+
+example-code: |
+  var feed = fc.data.feed.quandl()
+    .database('WIKI')
+    .dataset('AAPL')
+    .rows(10)
+    .descending(true)
+    .collapse('weekly');
+
+  // fetch some data!
+  feed(function(error, data) {
+    d3.select("#quandl")
+      .text(JSON.stringify(error ? error : data, null, 2));
+  });
+---
+
+Fetch data from the [Quandl API](https://www.quandl.com/docs/api#datasets). 
+The `database`, `dataset` must be specified to fetch data, by default it will retieve `YAHOO/GOOG`.
```

Comment:
The `database` and `dataset` specify which dataset is retrieved, by default it will retrieve `YAHOO/GOOG`.